x/nexus/client/cli/query.go

```diff
@@ -355,3 +356,36 @@ func GetCommandChainsByAsset() *cobra.Command {
 
 	return cmd
 }
+
+func GetCmdRecipientAddress() *cobra.Command {
```

Comment:
Why are these functions public?

Reply:
fair point, only reason is that all of them used to be public